primitives/src/currency.rs

```diff
@@ -225,9 +225,9 @@ pub type Lease = BlockNumber;
 pub enum DexShare {
 	Token(TokenSymbol),
 	Erc20(EvmAddress),
+	StableAssetPoolToken(StableAssetPoolId),
```

Comment:
wouldn't this cause issue with incentives pallet? I think we have some storages storing `CurrencyId::DexShare`